src/Adapters/EventSource.Netstandard13/Implementation/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="StringExtensions.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Text.RegularExpressions;
+
+namespace Microsoft.ApplicationInsights.EventSourceListener.Implementation
+{
+    internal static class StringExtensions
+    {
+        public static string WildCardToRegex(this string value)
```

Comment:
you do not need it any longer, do you?

Reply:
No, it is not needed. I'll remove the class.